pkg/smartcontract/rpcbinding/binding.go

```diff
@@ -284,6 +284,29 @@ func (res *{{toTypeName $typ.Name}}) FromStackItem(item stackitem.Item) error {
 {{- end}}
 	return nil
 }
+
+// ToStackItem creates [stackitem.Item] representing {{toTypeName $typ.Name}}.
+// It implements [stackitem.Convertible] interface.
```

Comment:
Then I'd suggest to introduce `var _ = stackitem.Convertible(&{{toTypeName $typ.Name}}{})` with corresponding comment (something about Convertible allowing to use this structure as an invocation argument).